app/config/config.yml

```diff
@@ -34,6 +35,8 @@ framework:
 twig:
     debug:            "%kernel.debug%"
     strict_variables: "%kernel.debug%"
+    globals:
+        google_map_api_key: AIzaSyBITmQHpnrjkl8CQOjd8x9JqgATOUESxEU
```

Comment:
I disagree as this parameter should remain the same over different install. Otherwise, each developer will have to have its very own google map api key.